File: Ex2/ex2c.c
#include <stdio.h>
int main()
{
  int year;
  printf("Enter the Year (YYYY) : ");
  scanf("%d",&year);
  if(year%4==0 && year%100!=0 || year%400==0)
    printf("\nThe Given year %d is a Leap Year", year);
  else
    printf("\nThe Given year %d is Not a Leap Year", year);

    return 0;
}

File: Ex2/ex2c.py
def main():
    year = int(input("Enter the Year (YYYY) : "))
    if (year % 4 == 0 and year % 100 != 0) or year % 400 == 0:
        print(f"\nThe Given year {year} is a Leap Year")
    else:
        print(f"\nThe Given year {year} is Not a Leap Year")

if __name__ == "__main__":
    main()
